src/stan/error_handling/matrix/check_ldlt_factor.hpp

```diff
@@ -28,9 +38,8 @@ namespace stan {
         msg << "is not positive definite. "
             << "last conditional variance is ";
```

Comment:
The last diagonal element of D in L * D * L' is the conditional variance of
the last variable given all the preceeding variables. Since this is a
variance, it cannot be negative if the covariance matrix is positive
definite.

On Mon, Jan 12, 2015 at 5:45 PM, Daniel Lee <notifications@github.com>
wrote:

> In src/stan/error_handling/matrix/check_ldlt_factor.hpp
> <https://github.com/stan-dev/stan/pull/1207#discussion-diff-22829542>:
>
> > @@ -28,9 +38,8 @@ namespace stan {
> >          msg << "is not positive definite. "
> >              << "last conditional variance is ";
>
> I don't know what this means.
> @bgoodri <https://github.com/bgoodri> or @mbrubake
> <https://github.com/mbrubake>, thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stan-dev/stan/pull/1207/files#r22829542>.
>